docs/hour-of-code-2022.html

```diff
@@ -115,6 +115,24 @@
 
 
         <script type="text/javascript">
+
+            function init(){
+                handleLangRedirects();
+            }
+
+            function handleLangRedirects(){
+                let language = navigator.language || navigator.userLanguage;
```

Comment:
Instead of auto redirect can you give a language option for these? This auto direct doesn't honor the language set in MakeCode either.

Autore direct is annoying if someone wants to see Portuguese in English browser or vice versa.